test/provisioner/common.go

```diff
@@ -0,0 +1,62 @@
+package provisioner
```

Comment:
Could you add a license header to this file?

See https://github.com/confidential-containers/cloud-api-adaptor/blob/main/test/provisioner/provision.go as for example:
```
// (C) Copyright Confidential Containers Contributors
// SPDX-License-Identifier: Apache-2.0
```